test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -715,7 +715,6 @@ Function sys.xid8rowversion(xid8,integer,boolean)
 Function sys.year(anyelement)
 Operator <=(sys."varchar",sys."varchar")
 Operator <=(sys.bpchar,sys.bpchar)
-Operator <>(sys."varchar",sys."varchar")
```

Comment:
Is this change expected?

Reply:
Yes. We are using <> operator in [linked_servers-vu-prepare](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_X_DEV/test/JDBC/expected/linked_servers-vu-prepare.out#L136) test file. This test was disabled until now.